text_generation/causal_lm/cpp/continuous_batching/python/tests/test_preemption.py

```diff
@@ -43,5 +64,5 @@ def test_preemption_with_multinomial(tmp_path, dynamic_split_fuse):
     model_path : Path = tmp_path / model_id
     save_ov_model_from_optimum(model, hf_tokenizer, model_path)
 
-    scheduler_config = get_scheduler_config({"num_kv_blocks": 3, "block_size": 32, "dynamic_split_fuse": dynamic_split_fuse, "max_num_batched_tokens": 256, "max_num_seqs": 256})
+    scheduler_config = get_scheduler_config({"num_kv_blocks": 30, "block_size": 32, "dynamic_split_fuse": dynamic_split_fuse, "max_num_batched_tokens": 256, "max_num_seqs": 256})
```

Comment:
let's keep current num blocks as is

for case n > 1 we need to compute precise amount of blocks to emulate preemption.

Reply:
Done